src/devices/machine/e05a30.cpp

```diff
@@ -167,6 +168,21 @@ void e05a30_device::write(offs_t offset, uint8_t data)
 	LOG("%s: e05a30_w([0xC0%02x]): %02x\n", machine().describe_context(), offset, data);
 
 	switch (offset) {
+		/* The documentation on the e05a30 in the LX-810/850 Technical Manual
+		 * is incomplete. There are instances of writing 0 to c000, so it is a guess
+		 * that writing to c000 will clear the shift register. */
+	case 0x00:
+		m_c000_shift_register = 0;
```

Comment:
You’re still missing a break here.  It’s falling through and setting the non-existent bits 24-31 of the shift register.